launch.go

```diff
@@ -61,6 +61,7 @@ type page struct {
 	CirSchemas              []surveys.CIMetadata
 	AccountServiceURL       string
 	AccountServiceLogOutURL string
+	SdsEnabled              string
```

Comment:
This might need to be a `bool` type instead?

Reply:
This cant be a bool because `settings.Get()` which is what is being used to get the env vars returns a string

Reply:
Yeah we're using standard library stuff to parse env vars as strings only atm :thinking: 

We could get around this by maybe having a struct to hold the different types? So like:
```
type Settings struct {
  StringSettings map[string]string
  BoolSettings map[string]bool
}
```
But then we would need to add different Getters and Setters for string type, and bool types.

I think we might want to avoid having string values for boolean values, though appreciate that this is a dev app, so maybe see what the others think? Using a Go package to do this sort of env parsing stuff for us might be useful, but might be out of scope for this work now?

Reply:
Yeah could be a good idea to have that discussion but I feel like anything we decide to do there is out of scope for this